internal/lock.go

```diff
@@ -86,9 +88,18 @@ func strToFileMode(perm string) (os.FileMode, error) {
 	return os.FileMode(parsed), nil
 }
 
+// Adds commas to number string at hundreds place, thousands place, etc.
```

Comment:
Maybe you could use https://github.com/dustin/go-humanize here. It would have the advantage of keeping the size of the output concise.